src/router/client.rs

```diff
@@ -232,14 +228,11 @@ impl RouterClient {
     }
 
     async fn handle_uplink(&mut self, logger: &Logger, uplink: Packet) -> Result {
-        self.store.store_waiting_packet(uplink)?;
-        // First uplink is used to get a quicker state channel connect than
-        // waiting for the state channel connect timer to trigger
-        if self.first_uplink {
-            self.first_uplink = false;
-            self.maybe_connect_state_channel(logger).await;
-        }
-        self.send_packet_offers(logger).await
+        // REVIEW: There was previously logic to handle first connection attempts.
+        // Is that still necessary, or will this just send packets into the void
+        // until a good connection can be made?
+        let packet = QuePacket::from(uplink);
+        self.send_packet(logger, Some(&packet)).await
```

Comment:
this isn't quite right.. you'd want to still queue packets if you can't reach router.. and then have a _send_packets_ that sends them all when it connects?